strawberry/experimental/pydantic/fields.py

```diff
@@ -42,6 +42,15 @@
     else:
         raise
 
+try:
+    from typing import Annotated
+except ImportError:
+    # python < 3.9 does not have Annotated
+    if sys.version_info < (3, 9):
+        Annotated = ()
+    else:
+        raise
```

Comment:
you can import it from typing extensions instead 😊

Reply:
Right!